### ISSUE TITLE
'hubert_pretraining' not in tasks.TASK_REGISTRY

### ISSUE THREAD

**Issue by Or-Tal** [facebookresearch/fairseq#3947]
## 🐛 Bug

Hi, 
I was trying to load the base pretrained hubert model taken from the examples section (smallest one)
It appears that the key 'hubert_pretraining' is missing in fairseq.tasks.TASK_REGISTRY.

the complete stacktrace is: 
![image](https://user-images.githubusercontent.com/47923357/136906221-b4930ca3-f7ee-423f-a838-9a20cea035fc.png)

### To Reproduce

I used the following code block (you could run it to reproduce error):

    from fairseq import checkpoint_utils

    class huBERT():
        def __init__(self, model_path, layer, device='cuda'):
            super().__init__()
            self.path = model_path
            self.layer = layer
            models, _ = checkpoint_utils.load_model_ensemble([self.path])  # <- this is where the code breaks
            self.model = models[0]
            self.model = self.model.to(device)
            for param in self.model.parameters():
                param.requires_grad = False

        def extract_feats(self, x):
            feats, _ = self.model.extract_features(source=x.squeeze(1), padding_mask=None, mask=False, output_layer=self.layer)
            return feats.detach()

    if __name__ == "__main__":
        model = huBERT(<path to hubert_base_ls960.pt>, 6)

#### Code sample
copy and run the above

### Expected behavior

I would expect it to retrieve a pretrained hubert model

### Environment

 - fairseq Version : 0.10.0
 - PyTorch Version: 1.9.0+cu102
 - OS (e.g., Linux): windows
 - How you installed fairseq (`pip`, source): pip install
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version: 10.2
 - GPU models and configuration:
 - Any other relevant information:

**Comment by wnhsu** [comment]
@Or-Tal Instead of running `pip install fairseq`, could you try installing from the source following the instruction [here](https://github.com/pytorch/fairseq#requirements-and-installation). I just tested and the latest commit works fine

**Comment by LaurinmyReha** [comment]
@wnhsu did not work for me unfortunately
